demos/http/http_demo_plaintext/http_demo_plaintext.c

```diff
@@ -160,21 +125,25 @@ static int32_t transportSend( HTTPNetworkContext_t * pContext,
 /**
  * @brief The transport receive function that defines the transport interface.
  *
+ * This is passed as the #HTTPTransportInterface.recv function used for reading
+ * data received from the network.
+ *
  * @param[in] pContext User defined context (TCP socket for this demo).
  * @param[out] pBuffer Buffer to read network data into.
  * @param[in] bytesToRead Number of bytes requested from the network.
  *
  * This is passed to the #HTTPTransportInterface.recv function and used to
```

Comment:
I noticed later that the sentence existed here as well :P.  Can you please remove this one?